next/components/HomePage/VerticalCardButton.tsx

```diff
@@ -0,0 +1,31 @@
+import ChevronRight from '@assets/images/chevron-right.svg'
+import Button, { ButtonProps } from '@modules/common/Button'
+import cx from 'classnames'
+
+type VerticalCardButtonProps = ButtonProps & {
+  size?: 'default' | 'medium' | 'large' | 'custom'
+}
+
+export const VerticalCardButton = ({
```

Comment:
For consistency, we usually export components as default at the end of the file. Please follow this pattern everywhere :)

Reply:
For naming conventions, what does `VerticalCardButton` stands for? Do we also have some horizontal card button?

Reply:
It stands for Button that takes up all vertical space that is available. No we don't have horizontal card button.